daos/customer.go

```diff
@@ -0,0 +1,56 @@
+package daos
```

Comment:
"daos" is not a golang-style name (it's came from Java), check golang-layouts repository